<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Diaporama</class>
 <widget class="QDialog" name="Dialog_Diaporama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Séléctionnez un diaporama à charger</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>28</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Diaporama</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rb_1">
          <property name="text">
           <string>Diaporama de Pantxika</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_2">
          <property name="text">
           <string>Diaporama de Thierry</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_3">
          <property name="text">
           <string>Diaporama de Yann</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_4">
          <property name="text">
           <string>Diaporama Manu</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_confirm">
       <property name="text">
        <string>Charger le diaporama</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
